lib/util.js

```diff
@@ -13,3 +14,17 @@ module.exports.tokens = function(options) {
     distroDescription: options.drupalDistro.description
   };
 };
+
+module.exports.npmVersion = function(name, relativeRepoUrl, useMaster) {
+  if (useMaster) {
+    return 'git+https:' + relativeRepoUrl + '#master';
+  }
+  else {
+    var version = spawn.spawnCommandSync(
+      'npm',
+      ['show', name, 'version'],
+      {stdio: 'pipe'}
+    ).stdout.toString('utf8');
+    return '^' + _.trim(version);
```

Comment:
Is this an intentional change from `~` currently used in the template package.json?